src/test/java/bio/terra/service/snapshot/flight/duos/UpdateSnapshotDuosFirecloudGroupIdStepTest.java

```diff
@@ -67,7 +65,7 @@ public void testDoAndUndoStepUnlink() throws InterruptedException {
     SnapshotLinkDuosDatasetResponse response =
         workingMap.get(JobMapKeys.RESPONSE.getKeyName(), SnapshotLinkDuosDatasetResponse.class);
     assertNotNull(response);
-    assertNull("No new DUOS dataset to link", response.getLinked());
+    assertNull(response.getLinked(), "No new DUOS dataset to link");
```

Comment:
Oh boy, they swapped the order of the arguments between versions of junit?

Reply:
Yes, they moved `message` to the end of the parameter list for all assertions. https://stackoverflow.com/a/50313815

My personal take is that, since we're using `assertThat` for some checks, we should use it for everything, as it supports everything that the assertions API does and more. So I would write `assertThat(x, nullValue())` instead of `assertNull()`.